src/rust-crypto/CrossSigningIdentity.ts

```diff
@@ -138,9 +139,21 @@ export class CrossSigningIdentity {
      * (If secret storage is *not* configured, we assume that the export will happen when it is set up)
      */
     private async exportCrossSigningKeysToStorage(): Promise<void> {
-        const exported: RustSdkCryptoJs.CrossSigningKeyExport = await this.olmMachine.exportCrossSigningKeys();
-        this.secretStorage.store("m.cross_signing.master", exported.masterKey!);
-        this.secretStorage.store("m.cross_signing.self_signing", exported.self_signing_key!);
-        this.secretStorage.store("m.cross_signing.user_signing", exported.userSigningKey!);
+        const exported: RustSdkCryptoJs.CrossSigningKeyExport | null = await this.olmMachine.exportCrossSigningKeys();
+        if (exported?.masterKey) {
```

Comment:
Unfortunately these aren't covered by tests, so the quality gate is failing.

You can try something like:

```suggestion
        /* istanbul ignore else (this function is only called when we know the olm machine has keys) */
        if (exported?.masterKey) {
```